cms/management/commands/create_initial_data.py

```diff
@@ -10,9 +10,21 @@ class Command(BaseCommand):
 
     help = 'Create initial data by using factories.'
 
-    def collect_initial_data_functions(self):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--app_label',
+            dest='app_label',
+            help='Provide an app label to create app specific data (e.g. --app_label boxes)',
+            default=False
+        )
+
+    def collect_initial_data_functions(self, app_label=None):
         functions = {}
-        for app in apps.get_app_configs():
+        if app_label:
+            app_list = [apps.get_app_config(app_label)]
```

Comment:
Could you please catch `LookupError` and print a user friendly message if there is no app named `app_label` (for example, "foo")?

We can use `self.stdout.write(self.style.ERROR('...'))` to print the message. I'm not a native English speaker so I left that part to you :)